src/SFA.DAS.AssessorService.Application.Api/Validators/ExternalApi/Certificates/GetBatchCertificateValidator.cs

```diff
@@ -26,48 +26,41 @@ public GetBatchCertificateRequestValidator(IStringLocalizer<GetBatchCertificateR
                 {
                     RuleFor(m => m).CustomAsync(async (m, context, cancellation) =>
                     {
-                        // NOTE: Currently we're making the Certificate & ILR record both mandatory
+                        // NOTE: Currently we're making the Certificate & ILR record both mandatory - this is wrong fixing it!
                         var requestedIlr = await ilrRepository.Get(m.Uln, m.StandardCode);
                         var sumbittingEpao = await organisationQueryRepository.GetByUkPrn(m.UkPrn);
+                        var existingCertificate = await certificateRepository.GetCertificate(m.Uln, m.StandardCode);
+                        var createdByCertificate = await certificateRepository.GetCertificateByOrgIdLastname(m.Uln, sumbittingEpao.EndPointAssessorOrganisationId, m.FamilyName);
 
-                        if (requestedIlr is null || !string.Equals(requestedIlr.FamilyName, m.FamilyName, StringComparison.InvariantCultureIgnoreCase))
-                        {
-                            context.AddFailure(new ValidationFailure("Uln", "Cannot find apprentice with the specified Uln, FamilyName & Standard"));
-                        }
-                        else if (sumbittingEpao is null)
+                        if (sumbittingEpao is null)
                         {
                             context.AddFailure(new ValidationFailure("UkPrn", "Specified UKPRN not found"));
                         }
                         else
                         {
                             var providedStandards = await standardService.GetEpaoRegisteredStandards(sumbittingEpao.EndPointAssessorOrganisationId);
```

Comment:
If we are stopping them from getting a certificate because they are not approved for that standard, should we also be validating on version as well?

Reply:
The UI does not prevent the search returning results when the EPAO is not approved for the version recorded in the certificate. The check is that the EPAO knows the ULN, FamilyName and Standard, the combination of which is deemed sufficient to verify their legitimate access to the data, and that they are approved on that Standard (unless the EPAO was approved in the past when the certificate was created!)
The message returned if at least one of these fields is not matched to a learner is
"Cannot find apprentice with the specified Uln, FamilyName & Standard"